src/wallet/walletutil.cpp

```diff
@@ -8,6 +8,7 @@
 #include <util/system.h>
 
 bool ExistsBerkeleyDatabase(const fs::path& path);
+bool ExistsSQLiteDatabase(const fs::path& path);
```

Comment:
5f17c73e8b890c92d2dc74e0e3bb34d217bc5160

Why add another declaration instead of `#include <wallet/sqlite.h>`?

Reply:
To avoid a circular dependency.

Reply:
> To avoid a circular dependency.

Which one? `test/lint/lint-circular-dependencies.sh` is green for me.

Reply:
I think there was one in a previous iteration of this PR. But I'm going to leave it as is for now.